handlers/caltrops.py

```diff
@@ -68,10 +68,13 @@ async def handle_message(self, client, message, state):
                     link, desc = random.choice(list(self.myth.items()))
                 elif target == 'random':
                     target = split[1]
-                    response = requests.get("https://en.wikipedia.org/api/rest_v1/page/random/summary").json()
-                    link = response["content_urls"]["desktop"]["page"]
-                    # shorten the blurb if it's over 250 characters
-                    desc = response["extract"] if len(response["extract"]) < 250 else response["extract"][:250] + "..."
+                    try:
+                        response = requests.get("https://en.wikipedia.org/api/rest_v1/page/random/summary").json()
```

Comment:
Consider using response.ok to determine if a response is good. Somewhat redundant given the try catch but it seems right